pkg/handlers/upgrade_policies.go

```diff
@@ -0,0 +1,271 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/ocm-agent/pkg/consts"
+	log "github.com/sirupsen/logrus"
+)
+
+type UpgradePoliciesHandler struct {
+	ocm       *sdk.Connection
+	clusterId string
+}
+
+// https://api.openshift.com/#/default/get_api_clusters_mgmt_v1_clusters__cluster_id__upgrade_policies
+func NewUpgradePoliciesHandler(o *sdk.Connection, clusterId string) *UpgradePoliciesHandler {
+	log.Debug("Creating new upgrade policies Handler")
+	return &UpgradePoliciesHandler{
+		ocm:       o,
+		clusterId: clusterId,
+	}
+}
+
+func (g *UpgradePoliciesHandler) AddUpgradePolicy(clusterID string, policy *cmv1.UpgradePolicy) (*cmv1.UpgradePolicy, error) {
+	log.Debugf("Sending add upgrade policy request to OCM API: %s %v", clusterID, *policy)
+	request := g.ocm.ClustersMgmt().V1().Clusters().Cluster(clusterID).UpgradePolicies().Add().Body(policy)
+
+	resp, err := request.Send()
+	if err != nil {
+		return nil, err
+	}
+	return resp.Body(), nil
+}
+
+func (g *UpgradePoliciesHandler) DeleteUpgradePolicy(clusterID string, upgradePolicyId string) (*cmv1.UpgradePolicy, error) {
+	log.Debugf("Sending delete upgrade policy request to OCM API: %s %s", clusterID, upgradePolicyId)
+	request := g.ocm.ClustersMgmt().V1().Clusters().Cluster(clusterID).UpgradePolicies().UpgradePolicy(upgradePolicyId).Delete()
+	_, err := request.Send()
+	if err != nil {
+		return nil, err
+	}
+	return nil, nil
+}
+
+func (g *UpgradePoliciesHandler) GetUpgradePolicies(clusterID string) ([]*cmv1.UpgradePolicy, error) {
+	var upgradePolicies []*cmv1.UpgradePolicy
+
+	log.Debugf("Sending get all upgrade polices request to OCM API: %s", clusterID)
+	collection := g.ocm.ClustersMgmt().V1().Clusters().Cluster(clusterID).UpgradePolicies()
+	page := 1
+	size := 100
```

Comment:
What if move it as var or const

Reply:
@samanthajayasinghe 
I tried to do this but the compiler didn't seem to like it. It seems that the spec allows `var (.....)